testdata/commands/admin/upgrade/extract/image-references/command.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+"${OC:-oc}" adm release extract --file=image-references quay.io/openshift-release-dev/ocp-release:4.11.0-x86_64
```

Comment:
Why we don't write unit tests to test this command's output is correct or not?

Reply:
it would be possible to inline all the stdout/stderr/exit-status capture and comparison in this script.  But presumably folks will have other things that also want that logic, so I've centralized it in `verify-commands.sh`.  We could convert to a `verify-commands.go` library, if that makes it more unit-test-y, but we still want to be executing an external `oc` command to get that end-to-end testing, and we had existing verification like `verify-generated-completions.sh` in shell, so :shrug:, it seemed easier to write the centralized code in shell.  But I don't feel too strongly about it, and I can port to the Go-library-calling-an-external-`oc`, if that's what `oc`-maintainers prefer.

Reply:
I checked the code and it would be difficult to add unit test case for `adm release extract` command without some refactorings. 

In my opinion, instead of adding a whole brand new step just for testing the output of this command, best thing is adding a e2e test case in here https://github.com/openshift/origin/blob/master/test/extended/cli/admin.go by skipping in disconnected. That will ensure that command works as expected?